### PR TITLE
explicitly include v1 module in jwplatform module and move version to…

### PR DESCRIPTION
**Pull request by polishmatt**
… avoid circular imports

### CODE REVIEW

**Comment by polishmatt** [comment]
FYI there also seem to be tests failing (including some of the new ones) when I run `make test`.

**Review by saurabhc123 — COMMENTED** [review]
Changes are good. Can all this be generalized in a way so that this could be reused across all projects that might need it. Also, there are two tests failing.

**Comment by polishmatt** [comment]
Not sure what you mean Saurabh. Using a config file instead of writing code _is_ the generalized solution. I'm actually going to revert that change though since it was only introduced last year and I'm worried about people using older versions of setuptools.

File: setup.py [hunk 1]
```diff
@@ -29,7 +29,6 @@ def get_version():
 
 setup(
     name='jwplatform',
-    version=get_version(),
```
Comment (sol1000) [inline setup.py]: `get_version()` should be updated to bring the version from the `version.py` or removed since there's no place calling this function

Comment (sol1000) [reply setup.py]: `__init__.py` should be updated to `version.py` at 
https://github.com/jwplayer/jwplatform-py/blob/58a225892ae19edf881417cab283626be362da63/setup.py#L22

**Review by sol1000 — APPROVED** [review]
Looks good to me 👍  Thanks @polishmatt for fixing the bug!!

**Pull request merged**